os/net/netdev/netdev_ioctl.c

```diff
@@ -414,6 +434,76 @@ static int ioctl_siocgifconf(FAR struct ifconf *ifc)
 	return ret;
 }
 
+/****************************************************************************
+ * Name: netdev_setipv6addr
+ *
+ * Description:
+ *   Set IPv6 address to netdev and join multicast group
+ *
+ * Parameters:
+ *   dev - NIC
+ *   inaddr - the address to set
+ *
+ * Return:
+ *
+ ****************************************************************************/
+static void netdev_setipv6addr(struct netif *dev, FAR const struct sockaddr_storage *inaddr)
+{
+	ip6_addr_t temp;
```

Comment:
Please add ip6_addr.h in lwip, in order to use this 'ip6_addr_t'.

Reply:
Thanks